src/main/java/org/craftercms/studio/impl/v2/service/policy/internal/PolicyServiceInternalImpl.java

```diff
@@ -51,6 +51,7 @@ public class PolicyServiceInternalImpl implements PolicyServiceInternal {
     public static final String CONFIG_KEY_STATEMENT = "statement";
     public static final String CONFIG_KEY_PATTERN = "target-path-pattern";
     public static final String CONFIG_KEY_PERMITTED = "permitted";
+    public static final  String CONFIG_KEY_DENIED = "denied";
```

Comment:
@sumerjabri @avasquez614 please confirm if the name is ok

Reply:
> @phuongnq the policy service is one of the few that has [unit tests](https://github.com/craftercms/studio/blob/develop/src/test/java/org/craftercms/studio/impl/v2/service/policy/internal/PolicyServiceInternalImplTest.java#L62), please make sure those are updated with the new changes too

Updated test case. Thanks!